pfrl/agents/goal_conditioned_td3.py

```diff
@@ -237,6 +238,11 @@ def update_q_func_with_goal(self, batch):
             if self.add_entropy:
                 next_log_prob = next_action_distrib.log_prob(next_actions_normalized)
                 entropy_term = self.temperature * next_log_prob[..., None]
+                if self.add_kl_regularization:
+                    kl_divergence = self.compute_kl(
```

Comment:
I think you need to compare `self.policy` and `self.prior_policy` since prior policy is the policy one step before

Reply:
Good point, that could be one solution, but I don't think that's good. 
It doesn't really matter which one to compare against. The epsilon value will just change. I would be better to compare against the stable policy, so that's why I chose to compare against target_plicy.

Reply:
I believed that we ran our experiments of KL divergence with the policy at the current and previous timestep though, right?